make.ps1

```diff
@@ -0,0 +1,235 @@
+		}
+	}
+	else
+	{	
+		echo "Unable to locate Git. The version will remain unchanged."
+	}
+	
+	if ($version -ne $null)
+	{
+		$mod = "mods/" + $modID + "/mod.yaml"
+		$replacement = (gc $mod) -Replace "Version:.*", ("Version: {0}" -f $version)
+		sc $mod $replacement
+
+		$prefix = $(gc $mod) | Where { $_.ToString().EndsWith(": User") }
+		if ($prefix -and $prefix.LastIndexOf("/") -ne -1)
+		{
+			$prefix = $prefix.Substring(0, $prefix.LastIndexOf("/"))
+		}
+		$replacement = (gc $mod) -Replace ".*: User", ("{0}/{1}: User" -f $prefix, $version)
+		sc $mod $replacement
+
+		echo ("Version strings set to '{0}'." -f $version)
+	}
+}
+
+function Test-Command
+{
+	if (Test-Path ./engine/OpenRA.Utility.exe)
+	{
+		$msg = "Testing " + $modID + " mod MiniYAML"
+		echo $msg
+		./engine/OpenRA.Utility.exe $modID --check-yaml
+	}
+	else
+	{
+		UtilityNotFound
+	}
+}
+
+function Check-Command
+{
+	if (Test-Path ./engine/OpenRA.Utility.exe)
+	{
+		echo "Checking for explicit interface violations..."
+		./engine/OpenRA.Utility.exe all --check-explicit-interfaces
+
+		$msg = "Checking for code style violations in OpenRA.Mods." + $modID + "..."
+		echo $msg
+		./engine/OpenRA.Utility.exe $modID --check-code-style OpenRA.Mods.$modID
+	}
+	else
+	{
+		UtilityNotFound
+	}
+}
+
+function Check-Scripts-Command
+{
+	if ((Get-Command "luac.exe" -ErrorAction SilentlyContinue) -ne $null)
+	{
+		echo "Testing Lua scripts..."
+		foreach ($script in ls "mods/*/maps/*/*.lua")
+		{
+			luac -p $script
+		}
+		echo "Check completed!"
+	}
+	else
+	{
+		echo "luac.exe could not be found. Please install Lua."
+	}
+}
+
+function Docs-Command
+{
+	if (Test-Path ./engine/OpenRA.Utility.exe)
+	{
+		Version-Command
+		./engine/OpenRA.Utility.exe all --docs | Out-File -Encoding "UTF8" DOCUMENTATION.md
+		./engine/OpenRA.Utility.exe all --lua-docs | Out-File -Encoding "UTF8" Lua-API.md
+	}
+	else
+	{
+		UtilityNotFound
+	}
+}
+
+function FindMSBuild
+{
+	$msBuildVersions = @("4.0")
+	foreach ($msBuildVersion in $msBuildVersions)
+	{
+		$key = "HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\{0}" -f $msBuildVersion
+		$property = Get-ItemProperty $key -ErrorAction SilentlyContinue
+		if ($property -eq $null -or $property.MSBuildToolsPath -eq $null)
+		{
+			continue
+		}
+		$path = Join-Path $property.MSBuildToolsPath -ChildPath "MSBuild.exe"
+		if (Test-Path $path)
+		{
+			return $path
+		}
+	}
+	return $null
+}
+
+function UtilityNotFound
+{
+	echo "OpenRA.Utility.exe could not be found. Build the project first using the `"all`" command."
+}
+###############################################################
+############################ Main #############################
+###############################################################
+if ($args.Length -eq 0)
+{
+	echo "Command list:"
+	echo ""
+	echo "  all             Builds the game, its development tools and the mod dlls."
+	echo "  version	        Sets the version strings for all mods to the latest"
```

Comment:
I guess you have a spaces vs tab issue, because the indentation doesn't look right for me:
![screen shot 2017-05-13 at 11 19 27](https://cloud.githubusercontent.com/assets/167819/26024682/5943aeca-37ce-11e7-8d1a-0eacd754ab27.png)